auth.py

```diff
@@ -21,6 +21,13 @@ def read_pwdb():
 def add_user(user, password, pwdb):
     if user not in pwdb:
         pwdb[user] = password
+    
+    elif user in pwdb:
```

Comment:
```suggestion
    else:
```
should be sufficient, since `user in pwdb` should always be true if `user not in pwdb` is false, right?